requirements.txt

```diff
@@ -28,7 +28,7 @@ python-dateutil==2.8.2
 python-rapidjson==1.8
 redis==4.3.4
 sentry-arroyo==2.17.6
-sentry-kafka-schemas==0.1.112
+sentry-kafka-schemas==0.1.113
```

Comment:
includes this change https://github.com/getsentry/sentry-kafka-schemas/pull/337/